lib/constants.dart

```diff
@@ -1,25 +1,22 @@
 import 'package:flutter/material.dart';
 
-const double sizedHeight = 150.0;
-const double subtitleTextFont = 30.0;
-const double generalTextFont = 20.0;
-const String titleApp = 'The movieDB APP';
-const double likeButtonSize = 35.0;
-const MaterialColor starsColor = Colors.yellow;
-const double genrePaddingHeight = 50.0;
-const double genrePaddingWidth = 100.0;
-const double subHeaderPosition = 150.0;
-const double posterRadius = 20.0;
-const double subHeaderPadding = 30.0;
-const String overviewTitle = 'Overview';
-const String genreTitle = "Genres";
-const double sizedBoxHeight = 200.0;
-const double favButtonSize = 35.0;
-const double movieTitleSize = 30;
-const double imagePosterWidth = 600;
-const double imagePosterHeigth = 240;
-const double imageAlignment = -1.0;
-const double generalPadding = 4.0;
-const double borderRadius = 8.0;
-const double boxDecorationWidth = 3.0;
-const double genresContainerHeight = 50;
+class Constants {
+  static const double sizedHeight = 150.0;
+  static const double movieTitleSize = 33.0;
+  static const double subtitleTextFont = 34.0;
+  static const double generalTextFont = 22.0;
+  static const double smallTextFont = 18.0;
+  static const String titleApp = 'The movieDB APP';
+  static const double likeButtonSize = 35.0;
+  static const MaterialColor starsColor = Colors.yellow;
```

Comment:
This value its not necessary to add it as constant. Colors.yellow its a constant itself. If you had used another format of color like Hex --> Color(0xFF00FF00); or RGB ---> Color.fromRGBO(38, 38, 38, 0.4); it would have been necessary to add it.